src/components/primitive/alerts/alert.component.tsx

```diff
@@ -1,6 +1,18 @@
-import { AlertContainer } from './alert.component.styles';
-import { AlertProps } from './alert.component.props';
+import { Text, IconSvg } from 'components/primitive';
+import { AlertComponentProps, TextColors, Variant } from './alert.component.props';
+import { getVariant, getColor } from './alert.component.utils';
 
-export const Alert: React.FC<AlertProps> = ({ variant, children }) => {
-  return <AlertContainer variant={variant}>{children}</AlertContainer>;
+export const Alert: React.FC<AlertComponentProps> = (props) => {
+  const { label, variant, icon } = props;
+
+  const BaseAlert = getVariant(variant ? Variant[variant] : undefined);
+  const BaseColor = getColor(variant ? TextColors[variant] : undefined);
```

Comment:
Why can't you decide on text color based on the alert variant? I am guessing there can be only one text color for a given variant.

Reply:
Also constants should be camel cased in javascript.

Reply:
It is using Box as an outer container and a Text component inside. text color was not inheriting. I have pushed the changes in the latest commit.